packages/pouchdb-checkpointer/src/index.js

```diff
@@ -35,6 +35,12 @@ function updateCheckpoint(db, id, checkpoint, session, returnValue) {
     if (returnValue.cancelled) {
       return;
     }
+
+    // if the checkpoint has not changed, do not update
+    if(doc.last_seq === checkpoint) {
```

Comment:
Tiny nit, we put a space here, gonna check why eslint isnt catching it but ill just fix on the way in